src/Invariants/drawing/draw_optimal_sets.jl

```diff
@@ -17,4 +17,28 @@ function draw_optimal_set(
         nodelabel=Graphs.vertices(g),
         layout=stressmajorize_layout
     )
+end
+
+function draw_optimal_set(
+    g::AbstractGraph,
+    optset::AbstractOptimalEdgeSet
+)
+
+    # initialize array of colors for each edge
```

Comment:
Let's try to have complete sentences for code comments (I know I might have missed this myself in many places), so this comment should be:

```julia
# Initialize an array of colors for each edge. 
```

Reply:
The same for the comments below